### PR TITLE
Added Get logged in user endpoint

### PR DESCRIPTION
**Pull request by lukasz-kkk**

### CODE REVIEW

File: backend/GymBeam/Controllers/UserController.cs [hunk 1]
```diff
@@ -6,6 +6,8 @@
 using GymBeam.Constants;
 using GymBeam.Queries;
 using MediatR;
+using System.Net;
```
Comment (tirey93) [inline backend/GymBeam/Controllers/UserController.cs:9]: Do we need this additional refs?

File: backend/GymBeam/Controllers/UserController.cs [hunk 2]
```diff
@@ -43,6 +47,35 @@ public ActionResult<UserResponse> Get(int id)
...
+        [HttpGet("LoggedIn")]
+#if !DEBUG
+        [Authorize(Roles = Roles.User)]
+#endif
+        public ActionResult<UserResponse> GetLoggedIn()
+        {
+            int userId;
+            try
+            {
+                string cookiesUserId = Request.Cookies["X-User-Id"];
```
Comment (tirey93) [inline backend/GymBeam/Controllers/UserController.cs:59]: Could you add this x user ID to constant and replace it everywhere? The same with other cookies names

File: backend/GymBeam/Controllers/UserController.cs (hunk 1 above)
Comment (lukasz-kkk) [reply backend/GymBeam/Controllers/UserController.cs:9]: It is necessary for HttpStatusCode (status code returned when an exception is caught)

File: backend/GymBeam/Constants/Cookies.cs [hunk 3]
```diff
@@ -0,0 +1,8 @@
+﻿namespace GymBeam.Constants
+{
+    public class Cookies
```
Comment (tirey93) [inline backend/GymBeam/Constants/Cookies.cs:3]: Public static

File: backend/GymBeam/Constants/Cookies.cs [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+    {
+        public const string UserId = "X-User-Id";
```
Comment (tirey93) [inline backend/GymBeam/Constants/Cookies.cs:5]: Public static

**Pull request merged**